ledger-core/v2/virtual/object/object.go

```diff
@@ -152,12 +157,16 @@ func (sm *SMObject) Init(ctx smachine.InitializationContext) smachine.StateUpdat
 		return ctx.Stop()
 	}
 
+	waitDuration := time.Second * time.Duration(sm.pulseSlot.PulseData().NextPulseDelta) / 10
```

Comment:
May be move 10% to config?

Reply:
@ivanshibitov which one? I knew only configuration.go, but it's more about components, not constants. I can move that to object SM constant

Reply:
Just move it to constant, for now

Reply:
I'd suggest to move that part to separate function (or method)